circle.yml

```diff
@@ -22,6 +22,7 @@ test:
   override:
     - yarn build
     - scripts/citests.sh:
+        timeout: 1200
         parallel: true
```

Comment:
What issue were you seeing here?

Reply:
I'm still debugging this, this is not the right solution though.

iphone7 tests were timing out (still are) I thought we had too many stories but looking at saucelabs it seems like it's not completing correctly and re-trying.

It may be related to the picturebook update

Reply:
Looks like the issue is related to the storybook update and iphone 7. I'll look into it separately https://github.com/gymnastjs/gymnast/issues/359